languages/common-lisp/exercises/concept/lists/lists-test.lisp

```diff
@@ -0,0 +1,69 @@
+;; Ensures that lists.lisp and the testing library are always loaded
+(eval-when (:compile-toplevel :load-toplevel :execute)
+  (load "lists")
+  (ql:quickload :fiveam))
+
+;; Defines the testing package with symbols from lists and FiveAM in scope
+;; The `run-tests` function is exported for use by both the user and test-runner
+(defpackage lists-test
+  (:use :cl :fiveam :lists)
+  (:export :run-tests))
+
+;; Enter the testing package
+(in-package :lists-test)
+
+;; Define and enter a new FiveAM test-suite
+(def-suite lists-suite)
+(in-suite lists-suite)
+
+(test new-list "Leslie needs a way to make a new empty list"
+      (is (equal '() (new-list))))
+
+(test list-of-things "Leslie needs to create a new list with three things on it"
+      (is (equal '(salt skquargzes butter)
+                 (list-of-things 'salt 'skquargzes 'butter))))
+
+(test adding-to-the-list "Leslie needs a way of adding items to a list"
+      (is (equal '(left-handed-frobz)
+                 (add-to-list 'left-handed-frobz (new-list))))
+      (is (equal '(left-handed-frobz butter)
+                 (add-to-list 'left-handed-frobz '(butter)))))
+
+(test peeking-at-the-list "Leslie needs a way to see what items are coming up on the list"
+      (let ((shopping-list '(left-handed-frobz salt skquargzes butter sananab
```

Comment:
Wow, I love this list <3

Reply:
I used up all my creativity on this one. I especially like "right handed macaroni"